### PR TITLE
A function to test if file exists

### LINKED ISSUE #9: Henge returns wrong exit code when fails
When passed docker-compose file doesn't exist, henge returns success exit code -  `0`

### PR DESCRIPTION
**Pull request by surajssd**
Before passing the file for parsing check if file exists added a function which checks if file exists and if it does checks if it is not a directory.

Fixes #9

### CODE REVIEW

**Comment by surajssd** [comment]
Seems like I need to rebase according to recent merge

File: henge.go [hunk 1]
```diff
@@ -28,11 +29,29 @@ func convertToVersion(objs []runtime.Object, version string) ([]runtime.Object,
 	return ret, nil
 }
 
+// Loop over a array of filepaths and check if it exists
+// if it exists check if it is not a directory.
+func ifFileExists(files []string) {
+	for _, filename := range files {
+		fileInfo, err := os.Stat(filename)
+		if err != nil {
+			glog.Fatalf("File %s not found.", filename)
```
Comment (kadel) [inline henge.go]: This prints also time and filename with line number, I don't think that user needs that.
You can use `fmt.Fprintf(os.Stderr, ....)` to print to stderr.

Comment (surajssd) [inline henge.go]: Yep that can be done!

**Comment by kadel** [comment]
LGTM

**Pull request merged**